docker/Makefile

```diff
@@ -0,0 +1,73 @@
+.PHONY: all help build_docker_base_1804 build_docker_base_l4t build run
+
+include .env
+export  # export all variables from .env
+
+export USERNAME=developer
+export HOST_UID=$(shell id -u)
+export CURR_DIR=$(notdir $(shell pwd))
+
+# UNLESS YOU ARE UPDATING THE MAKEFILE FUNCTIONALITY
+# This file should rarely need to be touched.
+
+# ADDING A NEW TARGET
+# Targets are defined in docker/docker-compose.yml (at the bottom, called "Services")
+# To add a new target:
+#     1. add the target to the docker compose file
+#     2. document the target option in the `all` rule in this file below
+
+all:
+	@echo "usage: make [COMMAND]"
+	@echo
+	@echo "COMMAND options:"
+	@echo "    help"
+	@echo "        - show this message"
+	@echo "    build_docker_base_[IMAGE_TYPE]"
+	@echo "        - build the base docker container"
+	@echo "        - IMAGE_TYPE options:"
+	@echo "            - 1804 (for most environments)"
+	@echo "            - l4t (for nvidia jetson)"
+	@echo "            - prebuilt (for jetson with Galactic + Pytorch)"
```

Comment:
update name to jetson_pytorch

Reply:
removed, cause it's no longer pointing to a different jetson dockerfile